config/locales/en/placements/pages.yml

```diff
@@ -1,5 +1,28 @@
 en:
   placements:
     pages:
-      feedback:
-        page_title: Feedback
+      start:
+        service: Manage school placements
+        related_content: Related content
+        guidance:
+          offer_trainee_teacher_placement: |
+            Guidance on what schools need to do to offer trainee teacher placements
+          how_to_find_a_trn: |
+            Guidance on how to find or request a TRN
+        content: |
+          This service is for schools offering placements for trainee teachers and 
+          Initial teacher training (ITT) providers in England 
+          (accredited ITT providers and lead partners).
+
+          If you are a school, use this service to record ITT placements at your organisation.
+
+          If you are an ITT provider, use this service to find ITT school placement opportunities.
+
+          ##Before you start##
+
+          If you are a school, you will be asked for:
+
+          - your mentors' Teacher reference number (TRN)
+          - your mentors' date of birth
```

Comment:
I'd like to see if we can try putting this Markdown content into a `.md` file rather than storing it in the locale files.

Let's chat about this, but I'm hoping we'll be able to have a file structure like this:

```
.
└── app/
    └── views/
        └── placements/
            └── pages/
                ├── show.html.erb
                ├── _landing-page.md
                ├── _privacy-policy.md
                └── _terms-and-conditions.md
```